datadog/data_source_datadog_sensitive_data_scanner_standard_pattern.go

```diff
@@ -35,10 +35,16 @@ func dataSourceDatadogSensitiveDataScannerStandardPattern() *schema.Resource {
 					Type:        schema.TypeString,
 					Computed:    true,
 				},
+				"description": {
+					Description: "Description of the standard pattern.",
+					Type:        schema.TypeString,
+					Computed:    true,
+				},
 				"pattern": {
-					Description: "Regex that the standard pattern applies.",
+					Description: "Regex to match, optionally documented for older standard rules. ",
 					Type:        schema.TypeString,
 					Computed:    true,
+					Deprecated:  "Refer to the description field to understand what the rule does.",
```

Comment:
Is it related to this?
Are you saying not all standard rules have a description?

Reply:
This matches the public documentation as well

<img width="479" alt="Screenshot 2024-12-05 at 16 54 59" src="https://github.com/user-attachments/assets/eeffca0f-f7da-440c-a9ab-0332262b89b4">